mobilecoind/clients/python/jupyter/wallet.ipynb

```diff
@@ -13,8 +13,9 @@
     "To run this notebook, make sure you have the requirements installed, and that you have compiled the grpc protos.\n",
     "\n",
     "```\n",
-    "cd mobilecoind/clients/python/mob_client\n",
-    "pip3 install -r requirements.txt\n",
+    "cd mobilecoind/clients/python/jupyter\n",
+    "./install.sh\n",
+    "cd ../lib/mobilecoin/\n",
```

Comment:
if you are using the package, you don't need to compile the protos since all of this is bundled with the pypi release

Reply:
Great - will fix up